tests/e2e/functional_test.go

```diff
@@ -30,12 +31,162 @@ type FunctionalSuite struct {
 	E2ESuite
 }
 
-// TODO
-func (s *FunctionalSuite) TestCreateGitSync() {
-	w := s.Given().GitSync("@testdata/gitsync.yaml").InitializeGitRepo().
+// GitSync testing for Numaflow with raw manifests
+func (s *FunctionalSuite) TestSimpleNumaflowGitSync() {
+
+	// create GitSync and initialize Git repo with initial manifest files
+	w := s.Given().GitSync("@testdata/gitsync.yaml").InitializeGitRepo("numaflow/initial-commit").
 		When().
 		CreateGitSyncAndWait()
 	defer w.DeleteGitSyncAndWait()
+
+	// verify Numaflow installation and pipeline creation
```

Comment:
is the "Numaflow installation" part of the comment outdated now?

Reply:
I suppose technically not since we're deploying the controller still. I will change the comment though since that isn't what we're looking to test for, just that the manifests are being applied.